specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2021-06-01-preview/examples/ServerList.json

```diff
@@ -0,0 +1,90 @@
+{
+  "parameters": {
+    "api-version": "2021-06-01-preview",
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "nextLink": "https://management.azure.com/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/providers/Microsoft.DBforPostgreSQL/flexibleServers?api-version=2021-06-01-preview&$top=3&$skiptoken=skiptoken",
+        "value": [
+          {
+              "sku": {
+                  "name": "Standard_D4s_v3",
+                  "tier": "GeneralPurpose"
+              },
+              "properties": {
+                  "fullyQualifiedDomainName": "c7d7483a8ceb.test-private-dns-zone.meru.com",
```

Comment:
Please make sure we don't have meru.com in this PR. meru.com is just for our onebox.